src/components/NavBar/NavBarLeftSide.tsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
 import Grid from "@material-ui/core/Grid";
 
-export const NavBarLeftSide = ({ children }) => {
+export interface INavBarLeftSideProps {
+  children: React.ReactChildren;
```

Comment:
no need for `children` on props, by default react/ts adds it. If the interface its empty then just remove it

Reply:
I get an error when I try to start/build the app
```bash
Binding element 'children' implicitly has an 'any' type.  TS7031
```